### ISSUE TITLE
prune matlab test output

### ISSUE THREAD

**Issue by grahamlopez** [project-asgard/asgard#135]
we need to wrap the matlab test.dat file writer to only update the dat files when something _other than the git hash_ has changed so that PRs are more meaningful, and more importantly, so that the size of this git repo doesn't grow too quickly.

**Comment by dlg0** [comment]
Done.

**Comment by grahamlopez** [comment]
I'd be curious to know (and have a short record) of the route that you chose to take. Can you describe it briefly here?

**Comment by dlg0** [comment]
Sure. 

- I've added the gold data files to the matlab repo. 
- I've added code to ensure they are always generated in the same relative directory within that repo.
- When the gold generation routines are run, they produce a temp file using the hash extracted from the existing files, then I diff the two files. If there is a difference (or if the file does not exsit), that particular file is updated with the present hash information.  

Note that I have not yet added a catch to ensure that the working git state is clean.
